packages/react-components/lib/item-unknown.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core';
+import ErrorIcon from '@material-ui/icons/Error';
+import { Typography, Grid } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  errorIcon: {
+    color: theme.palette.error.main,
+  },
+  errorMsg: {
+    margin: '0 0.5rem',
+  },
+  errorDisabled: {
+    pointerEvents: 'none',
+    filter: 'blur(.25rem)',
+  },
+}));
+
+export interface ItemUnknownProps {
+  errorMsg?: string;
+  showError: boolean;
+  children: JSX.Element | null;
+}
+
+export const ItemUnknown = (props: ItemUnknownProps): JSX.Element => {
```

Comment:
Recommend renaming this to something more meaning. `UnknownOverlay`, `ErrorOverlay` or something else that conveys what the component do.